users/vieting/experiments/switchboard/ctc/feat/baseline_args.py

```diff
@@ -237,6 +237,51 @@ def get_returnn_config(
         # network["source"] = specaug_layer_jingjing(in_layer=["features"])
         pass
 
+    if audio_perturbation and recognition:
+        feature_recog_net = copy.deepcopy(feature_net)
```

Comment:
```suggestion
        feature_net = copy.deepcopy(feature_net)
```
`feature_recog_net` is not used lateron. Also rename the usages below.